src/style.css

```diff
@@ -0,0 +1,93 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf") format("truetype");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf") format("truetype");
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir.ttc") format("truetype");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 20px 8px 0;
+}
+
+.search-bar {
+  position: relative;
+  margin-bottom: 20px;
+  width: 100%;
+}
+
+.search__input {
+  width: 100%;
+  color: #3d4e61;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.search__input:hover {
+  border: 1px solid #e1e7ed;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)